plugins/nodebb-plugin-caiz/languages/en-US/caiz.json

```
@@ -45,5 +90,82 @@
     "questions_description": "Feel free to ask questions or seek advice",
     "resources": "Resources & Information",
     "resources_description": "A place to share useful information and resources"
+  },
+  "members": {
```

Biome (2.1.2)

[error] The key members was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)